resources/zeebe.json

```diff
@@ -346,6 +346,24 @@
           "type": "Integer"
         }
       ]
+    },
+    {
+      "name": "ModelerTemplateIcon",
+      "superClass": [
+        "Element"
+      ],
+      "meta": {
+        "allowedIn": [
+          "zeebe:TemplateSupported"
```

Comment:
We could consider having this only for `bpmn:Task` for now, but I thought we can be that relaxed already.

Reply:
:+1: